cmd/syft/internal/constants.go

```diff
@@ -0,0 +1,6 @@
+package internal
+
+const (
+	Syft        = "syft"
+	NotProvided = "[not provided]"
```

Comment:
I'm not sure it's best to have these constants, or the right location for them.

Reply:
I think we should remove the syft one, could be convined to keep the not-provided one though (maybe renamed to something like `VersionMissing` or `EmptyVersion`)

Reply:
I got rid of the Syft constant here. I'm happy to rename the `NotProvided` variable, but I'd note it's used for more than just version:
```
	version        = internal.NotProvided
	buildDate      = internal.NotProvided
	gitCommit      = internal.NotProvided
	gitDescription = internal.NotProvided
```